### ISSUE TITLE
Missing Tag for v1.25.1

### ISSUE THREAD

**Issue by mtodor** [heiseonline/shariff#217]
As I see there is new release v1.25.1, but Tag is missing on github. That makes problem for some of our build systems.

In one case we are fetching zip file from github - and zip doesn't exist in this case:
https://codeload.github.com/heiseonline/shariff/zip/v1.25.1 or 
https://codeload.github.com/heiseonline/shariff/zip/1.25.1

In other case we are using composer with asset packagist and we are getting 1.25.1 over it:
https://asset-packagist.org/package/search?query=bower-asset%2Fshariff

As I see only problem is missing Tag on github, can you please investigate that?

**Comment by liayn** [comment]
The release was undone! There is no official 1.25.1 release.

**Comment by liayn** [comment]
No idea where bower takes this information from...

**Comment by mtodor** [comment]
Ok, I get it.

I tried with npm and same issue happens when I use asset packagist with npm:
https://asset-packagist.org/package/search?query=npm-asset%2Fshariff

But when I go to npm page, latest version is 1.24.1:
https://www.npmjs.com/package/shariff

It's really strange. It could be that asset-packagist doesn't fetch versions information properly.

What I have noticed with:
```
npm view shariff
```
There is version 1.25.1 listed in list of created versions: ```'1.25.1': '2017-04-21T08:11:33.129Z'```.
